datasetinsights/evaluation_metrics/metrics_utils.py

```diff
@@ -0,0 +1,47 @@
+import collections
+
+import numpy as np
+
+
+def mean_metrics_over_iou(metric_per_iou):
+    """Calculate mean value over IOUs.
+
+    Args:
+        metric_per_iou (dict): metric records for each iou
+
+    Returns (float):
+        mean metric values over IOUs
+    """
+    mean_sum = 0
+    for mean_ap in metric_per_iou:
+        result = mean_ap.compute()
+        mean_sum += np.mean(
+            [result_per_label for result_per_label in result.values()]
+        )
+    return mean_sum / len(metric_per_iou)
+
+
+def filter_pred_bboxes(pred_bboxes, max_detections):
+    """Save bboxes with same label in to a dictionary.
```

Comment:
The title string is not clear what this method is trying to achieve. Also, it should be super for users that `max_detection` applies to capped at the "top 100 detections ordered by decreasing detection score".